commons/src/main/java/org/frankframework/util/HttpUtils.java

```diff
@@ -76,4 +78,14 @@ public static String urlDecode(String input) {
 			throw new IllegalArgumentException(e);
 		}
 	}
+
+	public static String decodeBase64(String input) {
```

Comment:
Op andere plekken in de code doen we bv `byte[] versiondata = Base64.decodeBase64(versiondataB64);` . Heb je daar echt een nieuwe methode voor nodig?

Reply:
Ah, java util vs Apache commons 🤷🏻

Reply:
Oh, and DRY ofc. The try/catch is probs a good idea for stuff coming from the frontend.

Reply:
( Perhaps this can be moved to StringUtil or Misc? It doesn't really have much to do with http, and I want to remove the HttpUtils at some point.  )